### PR TITLE
docs(authz): Pipeline Permissions migratation

### PR DESCRIPTION
**Pull request by jervi**
Add documentation on using the automatic Pipeline Permissions migrator in front50.

Depends on https://github.com/spinnaker/front50/pull/476

### CODE REVIEW

File: setup/security/authorization/pipeline-permissions/index.md [hunk 1]
```diff
@@ -56,4 +56,35 @@ the previously selected service account in order to maintain backwards
 compatibility. Newly added triggers will use the permissions specified. If you
 wish to use pipeline permissions for the older triggers and already have
 permissions specified, edit your pipeline JSON and remove the `RunAsUser` field
-from your trigger.
+from your trigger, or enable the automatic migration (see next section).
+
+### Automatic migration
+
+Front50 can automatically migrate all pipelines from using Fiat service accounts to
```
Comment (dorbin) [inline setup/security/authorization/pipeline-permissions/index.md]: @jervi 
Maybe link "Fiat service accounts" to https://www.spinnaker.io/setup/security/authorization/service-accounts/ ?

**Comment by jervi** [comment]
@spinnaker/reviewers PTAL

**Review by dibyom — APPROVED** [review]

File: setup/security/authorization/pipeline-permissions/index.md [hunk 2]
```diff
@@ -56,4 +56,36 @@ the previously selected service account in order to maintain backwards
...
+new Pipeline Permissions. It will only run on pipelines where `roles` are not
+already present and `runAsUser` is set to a non-managed service account.
+
+The migration job will automatically create the new managed service users, and the
+new service user will get the same permissions as the manual service user that it
+replaces.
+
+{% include
+   warning
+   content="**Note:** If a pipeline has multiple triggers with different runAsUser's
```
Comment (dorbin) [inline setup/security/authorization/pipeline-permissions/index.md]: s/runAsUser's set/`runAsUser` fields set/

File: setup/security/authorization/pipeline-permissions/index.md [hunk 3]
```diff
@@ -56,4 +56,36 @@ the previously selected service account in order to maintain backwards
...
+
+### Automatic migration
+
+Front50 can automatically migrate all pipelines from using [Fiat Service
+Accounts](../service-accounts/) to use Pipeline Permissions and managed service
+accounts. The migrator is disabled by default, and can be enabled by setting
+the following flag:
+
+* Add the following to `front50-local.yml` (`~/.hal/default/profiles/front50-local.yml`
+if you're using Halyard):
```
Comment (dorbin) [inline setup/security/authorization/pipeline-permissions/index.md]: Suggested edit:

"..., and can be enabled by adding the following flag to `front50-local.yml`:

```yaml
migrations:
  migrateToManagedServiceAccounts: true
```
If you're using Halyard, the file is `~/.hal/default/profiles/front50-local.yml`."

Comment (jervi) [reply setup/security/authorization/pipeline-permissions/index.md]: Fixed

File: setup/security/authorization/pipeline-permissions/index.md (hunk 2 above)
Comment (jervi) [reply setup/security/authorization/pipeline-permissions/index.md]: Fixed

**Review by dorbin — APPROVED** [review]

**Comment by jervi** [comment]
Huh... The Travis build has completed, but the build status isn’t updated. https://travis-ci.org/spinnaker/spinnaker.github.io/builds/514359659

**Comment by dorbin** [comment]
@jervi 
Yeah, I've been noticing this with a lot of PRs lately. I'll give this some more time, and then merge it. The checks passed, so even if it doesn't get unstuck, I can safely force merge it.

**Comment by dorbin** [comment]
@jervi 
Could you push another commit--maybe just add a space or something--see if triggering a new build fixes this.

**Comment by robfletcher** [comment]
Travis seems to be circling the drain since being acquired by an asset stripping company recently. We're looking into other options.

**Comment by dorbin** [comment]
Thanks, Rob.

**Pull request merged**